ios/sdk/WeexSDK/Sources/Component/WXWebComponent.m

```diff
@@ -82,6 +82,7 @@ - (void)viewDidLoad
     _webview = (WXWebView *)self.view;
     _webview.delegate = self;
     _webview.allowsInlineMediaPlayback = YES;
+    _webview.scalesPageToFit = YES;
```

Comment:
what is this?

Reply:
@misakuo 

don't know, i just change the base branch to 0.16 - dev, this commit just appeared.

e83b0de is the last commit in dev branch, how to handle this situation ?

Reply:
maybe you can revert it simply

Reply:
How about close this PR, and create a new PR directly to 0.16-dev ?

Reply:
Sure, you can close this and create new PR based on 0.16-dev

Reply:
I see, thanks.